### ISSUE TITLE
HBLL re-evaluation

### ISSUE THREAD

**Issue by quang-huynh** [pbs-assess/dogfish-assess#9]
- Develop index without hook competition
- Look at standardization with Julian day (survey timing is getting earlier)

**Comment by seananderson** [comment]
Try the censored likelihood approach. https://cdnsciencepub.com/doi/full/10.1139/cjfas-2022-0159 `sdmTMB::censored_poisson()`  @jdunic can help :)

**Comment by quang-huynh** [comment]
@seananderson the choice of approach doesn't seem to affect the trend too much, so perhaps we proceed with the parsimonious model?

**Comment by seananderson** [comment]
I think the NB2 without hook competition is probably what we should go
with. Joe’s CJFAS paper discusses why the instantaneous catch rate method
is particularly bad for species that are unaffected by hook competition at
low saturation levels with dogfish as an example. Plus, when we look at the
CPUE vs hook saturation plots we see virtually no fall off of CPUE at
higher levels of saturation, even when we split by year. As a result, the
censored likelihood approaches end up not doing anything (what you saw
before was purely Poisson).

We had some problems with convergence once we added the censored likelihood
with the Poisson assuming there was some censoring at an arbitrary level,
although it’s possible that’s my fault given how I implemented the
likelihood.

I got a censored NB2 working yesterday, but that needs to be explored first
before we can use it in an assessment.
